native_modules/RNCookieManagerIOS.m

```diff
@@ -112,9 +112,11 @@ -(NSString *)getDomainName:(NSURL *) url
     NSInteger startPosition = count - maxLength;
 
     NSMutableString *result = [[NSMutableString alloc]init];
-    for (NSUInteger i = startPosition; i != endPosition; i++) {
-        [result appendString:separator];
-        [result appendString:[separatedHost objectAtIndex:i]];
+    for (NSInteger i = startPosition; i != endPosition; i++) {
```

Comment:
is not related to migrating the screens actually, but for some reason this was causing me issues with SSO login, not sure if it applies to current master, but if there are no reports, then it will stay here.